python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -178,7 +173,7 @@ def action_triggered():
                 menu_name = title
 
             self.desktop_window.add_project_command(
-                name, button_name, menu_name, icon, command_tooltip, groups
+                name, button_name, menu_name, icon, command_tooltip, groups, command_is_menu_default
```

Comment:
it's a pretty long line now. i suggest splitting it so we have a single param on each line!